code/modules/mapfluff/ruins/spaceruin_code/hilbertshotel.dm

```diff
@@ -7,6 +7,10 @@ GLOBAL_VAR_INIT(hhMysteryRoomNumber, rand(1, 999999))
 	icon_state = "hilbertshotel"
 	w_class = WEIGHT_CLASS_SMALL
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	//SKYRAT EDIT ADDITION - GHOST HOTEL UPDATE
+	var/list/static/hotel_maps = list("Generic", "Apartment")
```

Comment:
Add documentation to these two variables please.

Reply:
Good enough? (In recent commit)